damus/Util/Hashtags.swift

```diff
@@ -9,60 +9,70 @@ import Foundation
 import SwiftUI
 
 struct CustomHashtag {
-    let name: String
+    let imageName: String
+    let display: String
     let offset: CGFloat?
     let color: Color?
     
-    init(name: String, color: Color? = nil, offset: CGFloat? = nil) {
-        self.name = name
+    init(imageName: String, display: String, color: Color? = nil, offset: CGFloat? = nil) {
+        self.imageName = imageName
+        self.display = display
         self.color = color
         self.offset = offset
     }
-    
-    static let coffee = CustomHashtag(name: "coffee", color: DamusColors.brown, offset: -1.0)
-    static let bitcoin = CustomHashtag(name: "bitcoin", color: Color.orange, offset: -3.0)
-    static let nostr = CustomHashtag(name: "nostr", color: DamusColors.purple, offset: -2.0)
-    static let plebchain = CustomHashtag(name: "plebchain", color: DamusColors.deepPurple, offset: -3.0)
-    static let zap = CustomHashtag(name: "zap", color: DamusColors.yellow, offset: -4.0)
 }
 
-
 let custom_hashtags: [String: CustomHashtag] = [
-    "bitcoin": CustomHashtag.bitcoin,
-    "btc": CustomHashtag.bitcoin,
-    "nostr": CustomHashtag.nostr,
-    "coffee": CustomHashtag.coffee,
-    "coffeechain": CustomHashtag.coffee,
-    "plebchain": CustomHashtag.plebchain,
-    "zap": CustomHashtag.zap,
-    "zaps": CustomHashtag.zap,
-    "zapathon": CustomHashtag.zap,
-    "onlyzaps": CustomHashtag.zap,
+    "bitcoin": CustomHashtag(imageName: "bitcoin", display: "Bitcoin", color: Color.orange, offset: -3.0),
+    "bitcoinwallet": CustomHashtag(imageName: "bitcoin", display: "Bitcoin", color: Color.orange, offset: -3.0),
+    "bitcoinadoption": CustomHashtag(imageName: "bitcoin", display: "BitcoinAdoption", color: Color.orange, offset: -3.0),
+    "bitcoinjungle": CustomHashtag(imageName: "bitcoin", display: "BitcoinJungle", color: Color.orange, offset: -3.0),
+    "btc": CustomHashtag(imageName: "bitcoin", display: "BTC", color: Color.orange, offset: -3.0),
+    "coffee": CustomHashtag(imageName: "coffee", display: "Coffee", color: DamusColors.brown, offset: -1.0),
+    "coffeechain": CustomHashtag(imageName: "coffee", display: "CoffeeChain", color: DamusColors.brown, offset: -1.0),
+    "nostr": CustomHashtag(imageName: "nostr", display: "Nostr", color: DamusColors.purple, offset: -2.0),
+    "nostrica": CustomHashtag(imageName: "nostr", display: "Nostrica", color: DamusColors.purple, offset: -2.0),
```

Comment:
let's hold off on adding any new hashtags for now

Reply:
specifically want removing?